### ISSUE TITLE
Safari Footer Bug

### ISSUE THREAD

**Issue by itopaloglu83** [Joxit/docker-registry-ui#327]
Hello,

It seems like there is a bug related to how the footer element is rendered in Safari. It is somehow pushed down 20px forcing the Safari to always show a scrollbar. You can see the issue just by opening the app with Safari, no further steps are needed.

From what I can surmise, it's caused by two child components of the main overflowing 20px top and bottom. I experimented with my limited CSS knowledge and updating the `style.scss` seems to resolve the issue while keeping Chrome happy.

``` CSS
main {
  margin-top: 20px;
  margin-bottom: 20px;
  min-height: calc(100% - 156px);
}
```

I was going to create a pull request but wanted to start a discussion about it first in case I missed something.

**Comment by Joxit** [comment]
Hi, thank you for using my project and submitting issues 😊

Can you post one screenshot for the posterity?

**Comment by itopaloglu83** [comment]
Thank you for the project!

Before:
``` CSS
main {
  min-height: calc(100% - 136px);
}
```
<img width="1728" alt="Screenshot 2023-07-22 at 22 52 50" src="https://github.com/Joxit/docker-registry-ui/assets/461270/7cf30d1e-96d4-4a3f-bc92-8614b0e102b4">

After:
``` CSS
main {
  margin-top: 20px;
  margin-bottom: 20px;
  min-height: calc(100% - 156px);
}
```
<img width="1728" alt="Screenshot 2023-07-22 at 22 55 52" src="https://github.com/Joxit/docker-registry-ui/assets/461270/3669939c-1844-4075-94e5-cd79e6f7e2f9">

**Comment by Joxit** [comment]
Thank you for the details, this should be fixed in [v2.5.2](https://github.com/Joxit/docker-registry-ui/releases/tag/2.5.2)

I used flex instead of this old `min-height` to also support screens with small width.
